data/language/en-GB.txt

```diff
@@ -3665,6 +3665,27 @@ STR_6408    :Please install “innoextract” to extract GOG Installer, then res
 STR_6409    :The selected file is not the offline GOG Installer for RollerCoaster Tycoon 2. You may have downloaded the GOG Galaxy downloader stub or selected the wrong file.
 STR_6410    :Zoom in/out
 STR_6411    :Show buttons for zooming in and out in the toolbar
+STR_6412    :NumPad Return
```

Comment:
```suggestion
STR_6412    :NumPad Enter
```

The key on the numpad is always called Enter. (The Return key is above the right Shift key and also often called “Enter”.)